OpenRA.Mods.Common/Activities/UnloadCargo.cs

```diff
@@ -60,6 +60,12 @@ public override Activity Tick(Actor self)
 			if (IsCanceled || cargo.IsEmpty(self))
 				return NextActivity;
 
+			if (cargo != null && !cargo.CanUnload())
```

Comment:
`cargo` is already used twice by this point, so its guaranteed to not be null.

Reply:
Fixed.